docs/json.md

```diff
@@ -65,39 +61,11 @@ fetch the new row by id:
 {:id 1 :data {:some {:nested {:json 42}}}}
 ~~~
 
-Again, the JSON data returns as a Clojure map with no wrappers.
-
-When using JSON with HoneySQL though, some circs are still needed. Namely, you
-have to wrap a value with `[:lift ...]` as follows:
-
-~~~clojure
-(pgh/insert-one conn
-                :test_json
-                {:data [:lift {:another {:json {:value [1 2 3]}}}]})
-
-{:id 2, :data {:another {:json {:value [1 2 3]}}}}
-~~~
-
-Without the `[:lift ...]` tag, HoneySQL will treat the value as a nested SQL map
-and try to render it as a string, which will fail of course or lead to a SQL
-injection.
-
-Another way is to use HoneySQL parameters conception:
-
-~~~clojure
-(pgh/insert-one conn
-                :test_json
-                {:data [:param :data]}
-                {:honey {:params {:data {:some [:json {:map [1 2 3]}]}}}})
-~~~
-
-For details, see the [HoneySQL Integration](docs/honeysql.md) section.
+Again JSON returns as a Clojure map with no wrappers.
 
-PG2 supports only Clojure maps when encoding values into JSON. Vectors and other
```

Comment:
is this still true? because on top of this file you said both vectors and maps

if it is I'll return this ofc